alica_standard_library/include/Conditions.h

```diff
@@ -22,5 +27,10 @@ BOOST_DLL_ALIAS(alica_standard_library::AllChildFailure, AllChildFailure)
 BOOST_DLL_ALIAS(alica_standard_library::AlwaysTrueCondition, AlwaysTrueCondition)
 BOOST_DLL_ALIAS(alica_standard_library::AlwaysFalseCondition, AlwaysFalseCondition)
 BOOST_DLL_ALIAS(alica_standard_library::IsChildSuccess, IsChildSuccess)
+BOOST_DLL_ALIAS(alica_standard_library::IsEqualDouble, IsEqualDouble)
```

Comment:
What was suggested initially is that this looks something like
```suggestion
BOOST_DLL_ALIAS(alica_standard_library::IsEqual<double>, IsEqualDouble)
```

Reply:
Ohh yes,